### DISCUSSION TITLE
Thoughts on new shortcut for hurry

### DISCUSSION THREAD

**Discussion by makotech222** [induktio/thinker#81] [Ideas]
Hey, i really like the new shortcut to skip the damn confirmation window everytime. Some requests:

* Can we map it to something else? Ctrl+H is a pretty big stretch of my hand to hit on every city
* Can we add another shortcut to hurry for *all* bases, if it can be afforded? 

Thanks again, also love the new governor options you added recently.

**Comment by induktio** [comment]
In general keyboard remapping is not supported by the game engine and it would require A LOT more code being added to support that. It might be easier to just add support for mod shortcut remapping though. Hurry shortcut uses Ctrl+H because H is the shortcut for default dialog so it makes sense to use Ctrl for this. Also it should not be some combination that is pressed by accident because this shortcut can use a lot of credits. If the base has governor enabled, there's a separate option to enable production hurrying but again this only happens in some selected situations, not always for every base.

**Reply by makotech222** [reply]
yeah throwing hotkeys in the mod ini file would work fine
